Packs/CommonScripts/Scripts/ResolveShortenedURL/ResolveShortenedURL.py

```diff
@@ -1,44 +1,395 @@
+            url = response.headers["location"]
+            redirect_history.append(url)
+
+            try:
+                response = self._http_request(
+                    method="GET",
+                    full_url=url,
+                    resp_type="response",
+                    allow_redirects=False,
+                )
+
+            except Exception:
+                if not self.hit_redirect_limit(redirect_history):
+                    encountered_error = True
+                break
+
+        return URLUnshorteningData(
+            original_url=original_url,
+            resolved_url=url,
+            service_name=self.service_name,
+            redirect_history=[original_url] + redirect_history,
+            encountered_error=encountered_error,
+        )
+
+
+def unshorten_url(service_name: str, url: str, redirect_limit: int, session_verify: bool = True) -> CommandResults:
+    """
+    Unshorten a shortened URL.
+
+    Args:
+        service_name (str): The service to use for unshortening.
+        url (str): The URL to un-shorten.
+        session_verify (bool): Whether to verify the SSL certificate of the request.
+        redirect_limit (int): A maximum number of recursions to run. Use 0 for unlimited.
+    """
+    error_message: str = "There was an error while attempting to unshorten the final URL in the redirect chain.\n" \
+                         "It is possible that the unshortening process was not fully completed.\n\n"
+
+    service_class = URLUnshortingService.find_matching_service(service_name=service_name)
+    service_instance = service_class(redirect_limit=redirect_limit, verify=session_verify)
+    returned_data = service_instance.resolve_url(url=url)
+
+    readable_output = ""
+
+    if returned_data.encountered_error:
+        readable_output += error_message
+
+    readable_output += tableToMarkdown(name="URL Unshortening Results",
+                                       t=returned_data.to_hr_dict(),
+                                       headers=list(returned_data.to_hr_dict().keys()),
+                                       removeNull=True)
+
+    return CommandResults(
+        outputs_prefix="ResolveShortenedURL",
+        outputs_key_field="OriginalURL",
+        outputs=returned_data.to_context_dict(),
+        readable_output=readable_output,
+        indicator=Common.URL(url=returned_data.resolved_url,
+                             dbot_score=Common.DBotScore(
+                                 indicator=returned_data.resolved_url,
+                                 indicator_type=DBotScoreType.URL,
+                                 integration_name="ResolveShortenedURL",
+                                 score=Common.DBotScore.NONE,
+                             )),
+        raw_response=returned_data.raw_data,
+    )
+
+
+def main():  # pragma: no cover
+    default_service = "unshorten.me"
+    default_redirect_limit: int = 0  # Default value to use if `redirect_limit` is None
```

Comment:
Please put the defaults at the top of the file in capital letters
And change `default_redirect_limit` to `str`